active/marketing.md

```diff
@@ -0,0 +1,78 @@
+- Brand guidelines for Django and the DSF
+- KPIs aligned with objectives of the DSF
+
+Delegated responsibilities for the group to work on autonomously:
+
+- Management of the “www” djangoproject.com website content, in particular the Django blog and other content pages, excluding fundraising pages.
+- Management of one-off campaign sites like the [20th birthday site](https://birthday20.djangoproject.com/).
+
+Delegated responsibilities for the group to work on with others:
+
+- Management of fundraising pages on [djangoproject.com](http://djangoproject.com), in collaboration with the fundraising WG.
+- Contributions of marketing content to the “docs” [djangoproject.com](http://djangoproject.com) website content, in collaboration with Django core contributors.
+- Publication of new content across Django online community and social media channels, in collaboration with the Online Community WG and Social media WG.
+- Creation of physical and digital marketing collateral for Django events, and Django’s presence at events, in collaboration with the Events Support WG.
+
+## Initial membership
+
+- Chair: TBC
+- Co-Chair: TBC
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBC
+- Steering Council Liaison: TBC
+- Other members:
+  - Raffaella Suardini
+
+## Future membership
+
+We welcome everyone with an interest in marketing, and an understanding of the \[mission of the Django Software Foundation]. We select relevant volunteers based on our understanding of gaps in the group’s expertise.
+
+## ### How do people who want to join sign up / volunteer / express interest?
+
+To apply to join the group, fill our \[marketing working group application form].
+
+## ### How will decisions on adding/removing members be handled?
+
+Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members. The WG will vote for New Chair/Co-Chairs and decision to appoint will be based on gaining majority votes.
+
+Members join the group for one year term. At the end of this term, they need to opt into staying involved to keep being a member of the group.
+
+Members can be removed from the group with a 50%+1 vote from other members.
+
+### Access granting and deployment
+
+New members join the group with no access to the website’s content publishing functionality. After 6 months of onboarding, they can request publishing access to the Chair and Co-Chair, who decide whether the onboarding is completed, and if so liaise with the \[ops team] to manage permissions.
+
+For content managed via hard-coded templates within the website repository, contributions are made via the standard pull request process. Content changes are reviewed by other group members, with a final quality check by the \[website WG] before merging.
+
+## Budget
+
+No allocated budget. The group will likely request a budget to hire external marketing experts in the future.
```

Comment:
May be helpful to also include tools as a possible expenditure. I'm thinking something like figma

Reply:
~~I don't think it should call out Figma, but it can call out that the WG can spend money on tools. Or that this can be revisited on a as needed basis.~~

The no allocated budget line with an exception for hiring a consultant reads that there's no budget without a governance change. I guess I'm suggesting it be flexible.